pkg/crinit/init/init.go

```diff
@@ -117,36 +113,55 @@ type initClusterRegistryOptions struct {
 }
 
 func (o *initClusterRegistryOptions) Bind(flags *pflag.FlagSet, defaultServerImage, defaultEtcdImage string) {
-	flags.StringVar(&o.serverImage, "image", defaultServerImage, "Image to use for clusterregistry API server and controller manager binaries.")
-	flags.StringVar(&o.etcdImage, "etcd-image", defaultEtcdImage, "Image to use for etcd server.")
-	flags.StringVar(&o.etcdPVCapacity, "etcd-pv-capacity", "10Gi", "Size of persistent volume claim to be used for etcd.")
+	flags.StringVar(&o.serverImage, "image", defaultServerImage, "Image to use for the cluster registry API server binary.")
+	flags.StringVar(&o.etcdImage, "etcd-image", defaultEtcdImage, "Image to use for the etcd server binary.")
+	flags.StringVar(&o.etcdPVCapacity, "etcd-pv-capacity", "10Gi", "Size of the persistent volume claim to be used for etcd.")
 	flags.StringVar(&o.etcdPVStorageClass, "etcd-pv-storage-class", "", "The storage class of the persistent volume claim used for etcd. Must be provided if a default storage class is not enabled for the host cluster.")
-	flags.BoolVar(&o.etcdPersistentStorage, "etcd-persistent-storage", true, "Use persistent volume for etcd. Defaults to 'true'.")
-	flags.BoolVar(&o.dryRun, "dry-run", false, "dry run without sending commands to server.")
-	flags.StringVar(&o.apiServerOverridesString, "apiserver-arg-overrides", "", "comma separated list of clusterregistry-apisererver arguments to override: Example \"--arg1=value1,--arg2=value2...\"")
+	flags.BoolVar(&o.etcdPersistentStorage, "etcd-persistent-storage", true, "Use a persistent volume for etcd. Defaults to 'true'.")
+	flags.BoolVar(&o.dryRun, "dry-run", false, "Run the command in dry-run mode, without making any server requests.")
+	flags.StringVar(&o.apiServerOverridesString, "apiserver-arg-overrides", "", "Comma-separated list of clusterregistry-apisererver arguments to override: Example \"--arg1=value1,--arg2=value2...\"")
```

Comment:
s/clusterregistry-apisererver/clusterregistry-apiserver/

Or even better, just write it as  "cluster registry API server", the way it is written in other places.

Reply:
That makes sense. Done.